### PR TITLE
created a page for custom configuration of mindsdb

### LINKED ISSUE #2660: Create 'Custom mindsdb configuration strategies' page to include starting MindsDB locally with or without config file

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
Fixes #2660

## Please describe what changes you made, in as much detail as possible
  - created a new page, as below
image
image

mindsdb

### CODE REVIEW

File: docs/mindsdb-docs/docs/setup/custom-config.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Custom Configuration of MindsDB Locally
+
+To follow this guide, please make sure you have a local installation of MindsDB. [Here](https://docs.mindsdb.com/setup/self-hosted/pip/source/), you can find out how to install MindsDB locally.
+
+## Starting MindsDB with Default Configuration
+
+It is very straightforward to start MindsDB locally with the default config file - just run the commands below.
+
+```bash
```
Comment (ZoranPandovski) [inline docs/mindsdb-docs/docs/setup/custom-config.md:9]: Let's mention that users need to activate virtual env with this command

File: docs/mindsdb-docs/docs/setup/custom-config.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Custom Configuration of MindsDB Locally
```
Comment (ZoranPandovski) [inline docs/mindsdb-docs/docs/setup/custom-config.md:1]: Can we name this something like `Extend default mindsdb configuration` or similar?

File: docs/mindsdb-docs/docs/setup/custom-config.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        "type": "local"
+    },
+    "force_dataset_removing": false
+}
+```
+
+Now that your `config.json` file is ready, run the command below to start MindsDB locally with your custom configuration.
+
+```bash
+python -m mindsdb --config=config-file.json
```
Comment (ZoranPandovski) [inline docs/mindsdb-docs/docs/setup/custom-config.md:69]: Mention that --config flag is the location of config.json file

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: docs/mindsdb-docs/docs/setup/custom-config.md (hunk 1 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/setup/custom-config.md:9]: Done.

File: docs/mindsdb-docs/docs/setup/custom-config.md (hunk 2 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/setup/custom-config.md:1]: Done. Changed the navigation header and title.

File: docs/mindsdb-docs/docs/setup/custom-config.md (hunk 3 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/setup/custom-config.md:69]: I modified the command like this: `python -m mindsdb --config=/path-to-the-extended-config-file/config-file.json`

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**